notebooks/lscp_capacity.ipynb

```diff
@@ -0,0 +1,1129 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": true
+    }
+   },
+   "source": [
+    "# Location Set Covering Problem (LSCP)\n",
+    "\n",
+    "*Authors:* [Erin Olson](https://github.com/erinrolson),[Germano Barcelos](https://github.com/gegen07), [James Gaboardi](https://github.com/jGaboardi), [Levi J. Wolf](https://github.com/ljwolf), [Qunshan Zhao](https://github.com/qszhao)\n",
+    "\n",
+    "Location Set Covering is a problem realized by Toregas, et al. (1971). He figured out that emergency services must have placed according to a response time, since, there is a allowable maximum service time when it's discussed how handle an emergency activity. Therefore he proprosed a model named LSCP that:\n",
+    "\n",
+    "_Minimize the number of facilities needed and locate them so that every demand area is covered within a predefined maximal service distance or time._ Church L., Murray, A. (2018)\n",
+    "\n",
+    "**LSCP can be written as:**\n",
+    "\n",
+    "$\\begin{array} \\displaystyle \\textbf{Minimize} & \\sum_{j=1}^{n}{x_j} && (1) \\\\\n",
+    "\\displaystyle \\textbf{Subject to:} & \\sum_{j\\in N_i}{x_j} \\geq 1 & \\forall i & (2) \\\\\n",
+    "            & x_j \\in {0,1} & \\forall j & (3) \\\\ \\end{array}$\n",
+    "            \n",
+    "$\\begin{array} \\displaystyle \\textbf{Where:}\\\\ & & \\displaystyle i & \\small = & \\textrm{index referencing nodes of the network as demand} \\\\\n",
+    "& & j & \\small = & \\textrm{index referencing nodes of the network as potential facility sites} \\\\\n",
+    "& & S & \\small = & \\textrm{maximal acceptable service distance or time standard} \\\\\n",
+    "& & d_{ij} & \\small = & \\textrm{shortest distance or travel time between nodes} \\quad i \\quad \\textrm{and} \\quad j \\\\\n",
+    "& & N_i & \\small = & \\{j | d_{ij} < S\\} \\\\\n",
+    "& & x_j & \\small = & \\begin{cases} \n",
+    "          1, \\text{if a facility is located at node} \\quad j\\\\\n",
+    "          0, \\text{otherwise} \\\\\n",
+    "       \\end{cases} \\end{array}$\n",
+    "       \n",
+    "_This excerpt above was quoted from Church L., Murray, A. (2018)_\n",
+    "\n",
+    "This tutorial solves LSCP using `spopt.locate.coverage.LSCP` instance that depends on a array 2D representing the costs between facilities candidate sites and demand points. For that it uses a lattice 10x10 with simulated points to calculate the costs."
```

Comment:
Update intro with new description and reference/display new capacity formulation